system/greenhouse-ccloud/templates/logs-rt-pluginpreset.yaml

```
@@ -0,0 +1,114 @@
+{{- if .Values.openTelemetry.enabled -}}
+{{- range $r := $.Values.openTelemetry.clusters }}
+
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: otel-basic-auth-{{ $r.cluster }}
+  namespace: {{ $.Release.Namespace }}
+data:
+  failover_username_a: {{ required ".Values.openTelemetry.clusters[].openSearchLogs.failover_username_a missing" $r.openSearchLogs.failover_username_a | b64enc }}
+  failover_password_a: {{ required ".Values.openTelemetry.clusters[].openSearchLogs.failover_password_a missing" $r.openSearchLogs.failover_password_a | b64enc }}
+  failover_username_b: {{ required ".Values.openTelemetry.clusters[].openSearchLogs.failover_username_b missing" $r.openSearchLogs.failover_username_b | b64enc }}
+  failover_password_b: {{ required ".Values.openTelemetry.clusters[].openSearchLogs.failover_password_b missing" $r.openSearchLogs.failover_password_b | b64enc }}     
+{{- end }}
+
+---
+apiVersion: greenhouse.sap/v1alpha1
+kind: PluginPreset
+metadata:
+  name: logs
+  namespace: {{ $.Release.Namespace }}
+spec:
+  clusterSelector:
+    matchLabels:
+      greenhouse.sap/cluster-presets-enabled: "true"
```

This is the default now, please use this matcher from now on.

```suggestion
      greenhouse.sap/pluginpreset: "true"
```